brion/csvConfig.cpp

```diff
@@ -124,6 +103,26 @@ CsvConfig::CsvConfig(const URI& uri)
 {
 }
 
+size_ts CsvConfig::getNodeTypeIds() const
+{
+    size_ts output;
+    for (const auto& it : impl->nodeTypeIdToRowIndex)
+        output.push_back(it.first);
+    std::sort(output.begin(), output.end());
```

Comment:
If they are sorted, document it in the API, otherwise remove this and say they are not guaranteed to be in any order.